ros_queue_msgs/TransmissionVector command
ros_queue_msgs/TransmissionVector command_of_controller
---
float32 time_to_execute